lib/session.js

```diff
@@ -299,7 +299,7 @@ class Session {
     if (!stray.length) {
       return;
     }
-    cli.log(`Found stray commits in ${branchName}:\n` +
+    cli.log(`Found new commits in ${branchName}:\n` +
```

Comment:
This could also happen when the local branch contains commits not present in the upstream, I think `different` would be a more appropriate word?

```suggestion
    cli.log(`Found different commits in ${branchName}:\n` +
```

Reply:
"different" would confuse me, I'm not sure what they are different from. How about something like

> master doesn't match upstream/master

(subbing in the actual master branch name)

That seems both accurate and explicit.